js/startup/listingDelete.js

```diff
@@ -0,0 +1,56 @@
+import app from '../app';
+import { events as listingEvents } from '../models/listing';
+import Listing from '../models/listing/Listing';
+import Dialog from '../views/modals/Dialog';
+
+const getTitle = (model) => (
+  // Model might be a Listing or ListingShort instance.
+  model instanceof Listing ?
+    model.get('listing')
+      .get('item')
+      .get('title') :
+    model.get('title')
+);
+
+export default function () {
+  const statusMessages = {};
+
+  listingEvents.on('destroying', (model, opts) => {
+    let statusMessage = statusMessages[opts.slug];
+
+    if (statusMessage) return;
```

Comment:
Yeah, good catch. I forgot about that part. Should be fixed in a push coming soon.